en/drafts/originals/creating-guis-in-python-for-digital-humanities-projects.md

```diff
@@ -127,151 +93,102 @@ _Linux Users_: Use the following pip commands to install Qt Designer:
    path for the user to reach their goal within the application.
 
-The field of user interface design is now vast. The Additional Resources section below contains more advanced resources for
-those wishing to dive more deeply into the field.
+The field of user interface design is now vast. The [Additional Resources section](#additional-resources) below contains more advanced resources for those wishing to dive more deeply into the field.
 
 # A GUI Project: PDF Merger
 
-Merging PDF files in a directory is a relatively easy goal using Python and freely available packages. A basic script
-operated from the command line can accomplish the task. Yet, what if the user only desires to merge some of the PDFs in
-the directory rather than all of them? Or perhaps PDFs stored in various locations across the user’s hard drive need to
-be merged. In the former case, the user would need to remove the undesired files. In the latter case, the user would have
-to move or copy the files into one central location. An application with a GUI and drag-and-drop functionality can make
-the process trivial.
+Merging PDF files in a directory is a relatively easy goal to achieve using Python and other freely available packages. Equally, a basic script operated from the command line can accomplish the task. Yet, what if the user only desires to merge some of the PDFs in the directory rather than all of them? Or perhaps the user needs to merge PDFs stored in various locations across their hard drive. In the former case, the user would need to remove the undesired files. In the latter case, the user would have to move or copy the files into one central location. An application with a GUI and drag-and-drop functionality can make the process simple.
 
 ## Designing the Interface with Qt Designer
 
 To begin, launch the Qt Designer application you downloaded earlier by clicking on its icon. On MacOS and Windows the application will be located
-where you selected during the installation process; the default for MacOS is your Applications folders and the default for Windows is 
-C:\Program Files (x86)\Qt Designer. Linux users' path will vary but, for example on Ubuntu, they may find the application with the following commands:
+where you selected during the installation process; the default for MacOS is your Applications folders and the default for Windows is `C:\Program Files (x86)\Qt Designer`. Linux users' path will vary but, for example on Ubuntu, they may find the application with the following commands:
 
 `cd /usr/lib/x86_64-linux-gnu/qt5/bin/` 
 
 `./designer` 
 
 
-If you cannot locate the application, follow the instructions above in the Prerequisites and Tools section to download and install Qt Designer.
+If you cannot locate the application, follow the instructions above in the [Prerequisites and Tools section](#prerequisites-and-tools) to download and install Qt Designer.
```

Comment:
@DerDoktorFaust You'll notice that I've added in links to other sections of the lesson where you direct readers to them.

Reply:
Looks great, I like all the changes!